src/engraving/rw/read114/read114.cpp

```diff
@@ -271,9 +271,9 @@ static bool readTextProperties(XmlReader& e, ReadContext& ctx, TextBase* t, Engr
         t->setXmlText(s);
     } else if (tag == "foregroundColor") { // same as "color" ?
         e.skipCurrentElement();
-    } else if (tag == "frame") {
-        t->setFrameType(e.readBool() ? FrameType::SQUARE : FrameType::NO_FRAME);
-        t->setPropertyFlags(Pid::FRAME_TYPE, PropertyFlags::UNSTYLED);
+    } else if (tag == "border") {
```

Comment:
This line should not be changed (it needs to match what's seen in MuseScore 1 files)